src/MakeupContainer/MakeupContainer.js

```diff
@@ -52,15 +52,6 @@ const MakeupContainer = ({ id }) => {
           return makeup;
```

Comment:
Your `filterMakeup` function seems like it could be cleaned up a bit, but that doesn't need to be a priority now. I'm guessing it needs to be like that because the data you're getting from the API is not very consistent?

Reply:
Yes, exactly. I had it cleaned up quite a bit, but it was hitting the null and saying that it could run .toLowerCase() on null.

Reply:
makes sense! then I think this is fine!